crates/storage/db/src/abstraction/table.rs

```diff
@@ -48,7 +49,7 @@ pub trait Decompress: Send + Sync + Sized + Debug {
 /// Trait that will transform the data to be saved in the DB.
 pub trait Encode: Send + Sync + Sized + Debug {
     /// Encoded type.
-    type Encoded: AsRef<[u8]> + Into<Vec<u8>> + Send + Sync;
+    type Encoded: AsRef<[u8]> + Into<Vec<u8>> + Send + Sync + Ord + Debug;
```

Comment:
where is `Debug` used?

Reply:
`etl::Collector` needs it, and I don't see why not.